transfer-manager/contracts/RaribleTransferManager.sol

```diff
@@ -218,12 +217,17 @@ abstract contract RaribleTransferManager is OwnableUpgradeable, ITransferManager
     function calculateTotalAmount(
```

Comment:
could you add a description for the function?

Reply:
done